pocket_world_application/src/main/webapp/WEB-INF/views/city_resources.vm

```diff
@@ -39,10 +39,101 @@
     <div class="row">
         #foreach($position in [9,10,11,12])
             <div class="col-md-3">
-                #showbuilding($dto.getResourceBuildings(), $position )
+                        #showbuilding($dto.getResourceBuildings(), $position )
             </div>
         #end
     </div>
 
 </div>
+
+<script type="text/javascript" src="#springUrl('/js/jquery-2.1.4.min.js')"></script>
+<script type="text/javascript" src="#springUrl('/js/bootstrap.min.js')"></script>
+
+<script type="text/javascript">
+
+    $(document).ready(function(){
+
+        $("div[id*='square']").click(function(){
+            var $div = $(this);
+            var id_name = $(this).attr('id');
+            var $position = id_name.substr('square_'.length,id_name.length);
+            if($div.hasClass('building_empty')){
```

Comment:
you can use selector to avoid this if statement. for example, div[id*='square'].building_empty